tests/csm/rest/test_capacity_quota.py

```diff
@@ -90,12 +95,14 @@ def setup_method(self):
                                  "IAM user creation failed")
         self.uid = payload["uid"]
         self.user_id = resp.json()['tenant'] + "$" + self.uid
-        self.created_iam_users.add(self.user_id)
+        usr_val = resp.json()["keys"][0]
+        self.created_iam_users.update({usr_val['user']:usr_val})
```

Comment:
please test and revert if this not working.

Reply:
sure, will remove

Reply:
done